NARFO_BE/NARFO_BE/Models/User.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace NARFO_BE.Models
+{
+    public partial class User
+    {
+        public int UserId { get; set; }
+        public string Username { get; set; }
+        public string DisplayName { get; set; }
+        public string SalesRep { get; set; }
+        public string Name { get; set; }
+        public string Lastname { get; set; }
+        public DateTime? Registered { get; set; }
+        public string InceptionDate { get; set; }
+        public string Activated { get; set; }
+        public string ExpiryDate { get; set; }
+        public string AccountStatus { get; set; }
+        public string Tak { get; set; }
```

Comment:
is the spelling for Tak correct?

Reply:
Yes that the correct spelling, it is an abbreviation.